tutorials/abap-environment-rap-generator/abap-environment-rap-generator.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Generate a RAP Business Service With The Wizard
+description: Generate a RAP business service in SAP BTP ABAP Environment using the ABAP Repository Objects Wizard
+auto_validation: true
+time: 10
+tags: [ tutorial>beginner, software-product>sap-btp--abap-environment]
+primary_tag: programming-tool>abap-development
+---
+
+## Prerequisites
+- You have done one of the following:
+    - **Tutorial**: [Create an SAP BTP ABAP Environment Trial User](abap-environment-trial-onboarding)
+    - You have bought a licensed version of SAP BTP ABAP Environment
```

Comment:
Wrong indentation

Reply:
Hi Michael, This indentation is intentional. I discussed it with Daniel:
"- You have done one of the following:" is first level
    - **Tutorial**: [Create an SAP BTP ABAP Environment Trial User](abap-environment-trial-onboarding)
    - You have bought a licensed version of SAP BTP ABAP Environment
... are subordinate to "- You have done one of the following:" (ie they are second-level)
... whereas "- You have installed [ABAP Development Tools](https://tools.hana.ondemand.com/#abap), latest version" is first-level